draft-ietf-jose-hpke-encrypt.md

```diff
@@ -329,6 +329,13 @@ After verification:
 }
 ~~~
 
+# Key Management
+
+As noted in Section 9.2.3 of {{RFC9180}} reusing a single KEM key across multiple algorithm combinations MUST be avoided to maintain
+cryptographic security. Each key and its associated algorithm suite, comprising the KEM, KDF, and AEAD should be managed independently.
+This separation prevents unintended interactions or vulnerabilities between suites, ensuring the integrity and security guarantees of each
+algorithm suite are preserved.
+
```

Comment:
Section 9.2.3. of RFC9180 does not say anything about using a single key for multiple algorithms, it talks about using a single key for multiple modes (Base, Auth, PSK, AuthPSK). It notes that this should be safe, even if there is no proof this is the case.

Using multiple HPKE algorithm combinations with the same key is safe. However, using the same key with HPKE and non-HPKE is not generically safe. However, unless the non-HPKE mechanism is specifically designed to interact badly with HPKE, bad interactions are unlikely to be possible.

Reply:
I see reference to Section 9.2.3 of {{RFC9180}] is not correct but we would like to have this behavior to address the comment from Orie, see https://mailarchive.ietf.org/arch/msg/cose/Wb8ueEn71Tx0-ANh4ZdJjvYXJbQ/